pkg/controller/logstorage/users/users_controller.go

```diff
@@ -66,6 +66,7 @@ type UsersCleanupController struct {
 	scheme          *runtime.Scheme
 	esClientFn      utils.ElasticsearchClientCreator
 	elasticExternal bool
+	multiTenant     bool
```

Comment:
I don't understand why we need this - line 76 disables this controller unless `opts.MultiTenant` is true, so this is _always_ true, no?

Reply:
Yup, we are only calling this for multi tenancy.